components/removed_from_channel_modal/removed_from_channel_modal.tsx

```diff
@@ -2,19 +2,22 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import {Modal} from 'react-bootstrap';
 import {FormattedMessage} from 'react-intl';
 
-export default class RemovedFromChannelModal extends React.PureComponent {
-    static propTypes = {
-        currentUserId: PropTypes.string.isRequired,
-        onHide: PropTypes.func.isRequired,
-        channelName: PropTypes.string,
-        remover: PropTypes.string,
-    };
+type Props = {
+    currentUserId: string;
+    onHide: () => void;
+    channelName: string;
```

Comment:
We should add optional here given that `channelName` is not required

```suggestion
    channelName?: string;
```